### PR TITLE
Add GenericName and modify Categories fields

### PR DESCRIPTION
**Pull request by philipzae**

### CODE REVIEW

File: data/dev.tchx84.Portfolio.desktop.in [hunk 1]
```diff
@@ -4,7 +4,8 @@ Name=Portfolio
 Exec=dev.tchx84.Portfolio
 Terminal=false
 Type=Application
-Categories=Utility;
+GenericName=File Manager
+Categories=GTK;System;Utility;FileTools;FileManager;
```
Comment (tchx84) [inline data/dev.tchx84.Portfolio.desktop.in:8]: We should probably stick to one main category :)

```
 desktop-file-validate /var/lib/flatpak/app/dev.tchx84.Portfolio/current/active/files/share/applications/dev.tchx84.Portfolio.desktop

/var/lib/flatpak/app/dev.tchx84.Portfolio/current/active/files/share/applications/dev.tchx84.Portfolio.desktop: hint: value "GTK;System;Utility;FileTools;FileManager;" for key "Categories" in group "Desktop Entry" contains more than one main category; application might appear more than once in the application menu
```

**Review by tchx84 — CHANGES_REQUESTED** [review]

File: data/dev.tchx84.Portfolio.desktop.in (hunk 1 above)
Comment (philipzae) [reply data/dev.tchx84.Portfolio.desktop.in:8]: Is FileManager the preferred category?

Comment (tchx84) [reply data/dev.tchx84.Portfolio.desktop.in:8]: Yes, it's exactly what it is :)

Comment (AsciiWolf) [reply data/dev.tchx84.Portfolio.desktop.in:8]: This is what Nautilus has in its desktop file:
`Categories=GNOME;GTK;Utility;Core;FileManager;`

So, correct categories for Portfolio should be:
`Categories=GTK;Utility;FileManager;`

Also, consider adding some keywords as well.For example, based on Nautilus:
```
# TRANSLATORS: Search terms to find this application. Do NOT translate or localize the semicolons! The list MUST also end with a semicolon!
Keywords=folder;manager;explore;disk;filesystem;
```

File: data/dev.tchx84.Portfolio.desktop.in [hunk 2]
```diff
@@ -4,7 +4,8 @@ Name=Portfolio
 Exec=dev.tchx84.Portfolio
 Terminal=false
 Type=Application
-Categories=Utility;
+GenericName=File Manager
+Categories=FileManager;
```
Comment (AsciiWolf) [inline data/dev.tchx84.Portfolio.desktop.in:8]: No, this is also incorrect. Additional categories should always be used in combination with the main category, not alone. You should ideally use what I suggested in my previous comment: `Categories=GTK;Utility;FileManager;`

**Review by AsciiWolf — CHANGES_REQUESTED** [review]

File: data/dev.tchx84.Portfolio.desktop.in (hunk 2 above)
Comment (philipzae) [reply data/dev.tchx84.Portfolio.desktop.in:8]: @tchx84 said to use a single category in Categories.

Comment (tchx84) [reply data/dev.tchx84.Portfolio.desktop.in:8]: I said one main category :)

Comment (AsciiWolf) [reply data/dev.tchx84.Portfolio.desktop.in:8]: @philipzae Could you please edit your last commit and change the Categories to `GTK;Utility;FileManager;`? Thanks!

Comment (philipzae) [reply data/dev.tchx84.Portfolio.desktop.in]: Done

Comment (AsciiWolf) [reply data/dev.tchx84.Portfolio.desktop.in]: Thanks, LGTM now. :-)

Comment (tchx84) [reply data/dev.tchx84.Portfolio.desktop.in]: Thanks !!

**Comment by philipzae** [comment]
Pull #48 was closed for this pull, but this one doesn't have keywords in it.

**Comment by AsciiWolf** [comment]
Actually, there already are keywords in the desktop file. I have overlooked them in my previous comments, so sorry for the noise. :-)

**Comment by philipzae** [comment]
So can the patch in #48 get pushed in?

**Comment by AsciiWolf** [comment]
Why? The keywords it adds are either redundant or not relevant. The current keywords are the same as used by Nautilus.

**Comment by AsciiWolf** [comment]
Why? The keywords it adds are either redundant or not relevant. The current keywords are the same as used by Nautilus.

**Pull request merged**